ReportSimulationOutput/measure.rb

```diff
@@ -872,6 +919,82 @@ def get_outputs(runner, args)
       @totals[TE::Net].timeseries_output = @totals[TE::Total].timeseries_output.zip(outputs[:elec_prod_timeseries]).map { |x, y| x + y * unit_conv }
     end
 
+    # Resilience
+    @resilience.each do |key, resilience|
+      next unless key == RT::Battery
+
+      resilience.variables.map { |v| v[0] }.uniq.each do |sys_id|
```

Comment:
I don't think this code would actually work if there were multiple batteries, since the `sys_id` is not showing up in the output at all. It looks like the last battery calculation would just override the previous ones. I guess nothing necessarily has to be done in the code here since it gives the correct output, but it is/will be confusing if we ever do support multiple batteries.

Reply:
Should we therefore leave this as it is now, until 2+ batteries are allowed? Or what changes, specifically, are you suggesting?

Reply:
I guess it's fine to leave for now. We'd need to refactor the code a bit if we ever allow multiple batteries or additional resilience outputs, but that can happen in the future. I'll mark this as resolved.